sec/review.tex

```diff
@@ -20,29 +21,32 @@ \subsection{Clustering}\label{subsec:clustering}
 Clustering originated in the social sciences around the turn of the
 20\textsuperscript{th} century in a similar manner to several other (now
 ubiquitous) statistical tools, including hypothesis testing and \(p\)-values,
-correlation, and factor analysis. Early work on cluster analysis, such as Driver
-and Kroeber~\cite{Driver1932} and Cattell~\cite{Cattell1943}, focused on its
-applications in attempting to identify traits which led to cultural and
-psychological differences between groups of people, as opposed to studying the
-methods used to identify any partitions.
+correlation, and factor analysis. Early work on cluster analysis, such
+as~\cite{Cattell1943,Driver1932}, focused on its applications in attempting to
+identify traits which led to cultural and psychological differences between
+groups of people, as opposed to studying the methods used to identify any
+partitions.
 
 The focus of clustering research shifted to the statistical nature of the
-methods following work by Sokal~\cite{Sokal1966} (later revised with
-Sneath~\cite{Sneath1973}) on clustering under the name \emph{numerical
-taxonomy}. These works were highly influential and led to an abundance of
-research into clustering methods, including~\cite{Diday1976}
-and~\cite{Hartigan1975} which each formalised the principles of clustering as a
-part of statistics. Furthermore, this period witnessed the advent of seminal
-work on clustering algorithms such as \(k\)-means~\cite{Hartigan1979} and
-hierarchical clustering~\cite{Defays1977,Sibson1973} that are still considered
-fundamental methods. Since then, with an increase in the data under study,
-clustering has fallen more squarely into the category of machine learning
-techniques that perform \emph{unsupervised learning} --- this is because
-clustering relies solely on the entirety of the observed data and some a priori
-knowledge such as a set of parameters~\cite{Dayan1999}.
+methods following work on clustering under the name \emph{numerical
+taxonomy}~\cite{Sneath1973,Sokal1966}. These works were highly influential and
+led to an abundance of research into clustering methods,
+including~\cite{Diday1976} and~\cite{Hartigan1975} which each formalised the
+principles of clustering as a part of statistics. Furthermore, this period
+witnessed the advent of seminal work on clustering algorithms such as
+\(k\)-means~\cite{Hartigan1979} and hierarchical
+clustering~\cite{Defays1977,Sibson1973} that are still considered fundamental
+methods.
+% TODO Can we have a reference or 2 here to backup claim that they're
+% still fundamental? (Perhaps even the initialisation paper of yours.
```

Comment:
I'm not sure if the initialisation paper is a good fit here. Maybe a catch-all clustering literature review or textbook reference?